### DISCUSSION TITLE
How to update label of an exsiting workflow

### DISCUSSION THREAD

**Discussion by journey-wang** [argoproj/argo-workflows#7779] [Q&A] [answered]
Hello everyone, I've create a workflow using argo go sdk, and add some labels to the workflow.
But I cannot find a way to update the label's value via exising argo sdk APIs.
Please help!

**Comment by terrytangyuan** [comment, marked as answer]
This isn't supported via the existing SDK (tracked in https://github.com/argoproj/argo-workflows/issues/5464) but you should be able to patch the labels using K8s Go SDK directly.
